Makefile

```diff
@@ -1,5 +1,10 @@
-IMG ?= "crossplane/stack-minimal-gcp"
-VERSION ?= "0.2.1"
+STACK_VERSION ?= 0.2.1
```

Comment:
Not blocking, but does this version need to be provided? In a worst case it causes the old version to be rebuilt and republished.  In the best case it happens to get the version correct until the next release and requires updates on each release.

What harm is there in requiring this value be provided?

Reply:
Do you mean provided to the `make publish` command, or provided to the pipeline job? I could see making it required for the job, but I'm a bit more skeptical about making it provided for the `make` command.